service-workers/service-worker/iframe-controller.https.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<title>Service worker inheritance</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="resources/test-helpers.sub.js"></script>
+<script>
+
+promise_test(function(t) {
+    var script = 'iframe-controller-worker.https.js';
+    var scope1 = 'iframe-controller.https.html';
+    var scope2 = 'resources/blank.html';
+    var script1 = script + '?for_scope1';
+    var script2 = script + '?for_scope2';
+    var registration1, registration2;
+    var parent_controller, child_controller;
```

Comment:
Why did you declare child_controller at this scope? I think just declaring it in all the closures where you actually use it would be cleaner (I don't think you ever use it across closure boundaries?)